cmeutils/plot_tools.py

```diff
@@ -0,0 +1,21 @@
+import scipy.signal as signal
+
+
+def find_peaks(data):
```

Comment:
I think this function would be a good candidate to accept a key word arguments (kwargs) parameter that is passed into the `signal.find_peaks()` function. [This tutorial](https://www.pythontutorial.net/python-basics/python-kwargs/) seems like a good place to start if you're not already familiar with `kwargs` in python.